src/libs/ajax.js

```diff
@@ -411,7 +412,10 @@ export const Jupyter = {
 
     return {
       create: clusterOptions => {
-        return fetchLeo(root, _.mergeAll([authOpts(), jsonBody(clusterOptions), { method: 'PUT' }]))
+        const body = _.merge(clusterOptions, {
+          labels: { saturnAutoCreated: 'true', saturnVersion: version }
```

Comment:
I don't know if we are ever going to want to create clusters in other contexts, but if so they presumably wouldn't have the 'saturnAutoCreated' property.

Reply:
Definitely, but for the time being we were doing this in 3 places so I wanted to centralize it.